core/src/main/scala/scala/pickling/internal/DefaultRuntimePicklerGenerator.scala

```diff
@@ -21,7 +20,9 @@ import scala.reflect.runtime.{universe => ru}
 [error] 	scala.pickling.array.binary.ArrayBinaryTest
 [error] 	scala.pickling.test.runtime.array.RuntimeArrayTest
  */
-class DefaultRuntimePicklerGenerator(reflectionLock: ReentrantLock) extends spi.RuntimePicklerGenerator {
+class DefaultRuntimePicklerGenerator(reflectionLock: ReentrantLock)
```

Comment:
Just an FYI, I'll be altering this interface before 0.11.x goes out.  Still working out the kinks locally.